### DISCUSSION TITLE
Welcome to dace Discussions!

### DISCUSSION THREAD

**Discussion by tbennun** [spcl/dace#837] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by definelicht** [comment]
I'm responsible for the FPGA backend along with @TizianoDeMatteis 🤓 I also designed and implemented the concept of Library Nodes
